router-bridge/js-src/plan.ts

```diff
@@ -281,7 +282,9 @@ export class BridgeQueryPlanner {
     let result = new Map<string, string>();
 
     subgraphs.names().forEach((name) => {
-      let sdl = printSchema(subgraphs.get(name).schema.toGraphQLJSSchema({}));
+      let sdl = printSchemaWithDirectives(
```

Comment:
You don't have to use `graphql-tools` here, or call `toGraphQLJSSchema()`. There's a `printSchema()` function in `@apollo/federation-internals` (code [here](https://github.com/apollographql/federation/blob/6b52c60d4ee9c77fee832d119d9ef752f51e16e6/internals-js/src/print.ts#L94)), if you import that you can just use `printSchema(subgraphs.get(name).schema)`.

Reply:
Updated to use the one from fed internals in [4ff041b](https://github.com/apollographql/federation-rs/pull/568/commits/4ff041b6f9936a9e45b2eabead1194b4b57e7a07)